README.rst

```diff
@@ -378,6 +378,12 @@ The following options (``[flags]``) are supported:
 * ``--name string`` defines the application name.
   By default, it is taken from the application rockspec.
 
+* ``--timeout string`` Time to wait for instance(s) start in background.
+  Can be specified in seconds or in the duration form (``72h3m0.5s``).
```

Comment:
`72h3m0.5s`

`0.5s`

Is it really possible to specify a half of second?

Reply:
Yes, it's parsed to `500 * time.Milisecond`. (but I will check id additionally)

Reply:
Yes, it's parsed to `500ms`. I've checked.